### ISSUE TITLE
Chalk missing from dependencies in Sharp branch

### ISSUE THREAD

**Issue by jgillman** [jimmynicol/image-resizer#41]
For pull #31:

I'm getting the following error with a totally fresh install and running `gulp watch`:

```
[gulp] [nodemon] starting `node index.js`
module.js:338
    throw err;
          ^
Error: Cannot find module 'chalk'
    at Function.Module._resolveFilename (module.js:336:15)
    at Function.Module._load (module.js:278:25)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (/Users/jgillman/Code/pantograph/index.js:14:15)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Function.Module.runMain (module.js:501:10)
    at startup (node.js:129:16)
    at node.js:814:3
[gulp] [nodemon] app crashed - waiting for file changes before starting...
```

Looks like `chalk` wasn't previously used in `index.js`. Adding it as a dependency in `bin/image_resizer.js` should solve it.

This does not crash the app when it's deployed to Heroku.

**Comment by jimmynicol** [comment]
This should be fixed in the latest push to `sharp` branch.

**Comment by jgillman** [comment]
Awesome, thanks. (fixed with dfa30bcfeb08a96d6b317d9410d11d68a30200a0)
